wkhtmltopdf/docker-entrypoint.sh

```diff
@@ -1,4 +1,4 @@
 #!/bin/sh
-service cron restart
+crond -l 2
```

Comment:
Is this essentially doing the same thing as before or does 2 correspond to a corn period?

Reply:
Yeah this is basically the log level for crond which only alerts on critical alerts. crond is the alpine version of the cron daemon. service isn't a thing in alpine. Alpine containers ahve their own cron daemon that can be activated by default and the standard folders it picks up from are different too, hence the change to those folders :)

I tested this locally by adding a script in for it to run every 15 mins and checked it was working. It runs the crond as a sub process of the main .sh process